src/plan/sticky/immix/global.rs

```diff
@@ -41,7 +40,7 @@ pub struct StickyImmix<VM: VMBinding> {
 
 /// The plan constraints for the sticky immix plan.
 pub const STICKY_IMMIX_CONSTRAINTS: PlanConstraints = PlanConstraints {
-    moves_objects: crate::policy::immix::DEFRAG || crate::policy::immix::PREFER_COPY_ON_NURSERY_GC,
+    moves_objects: true,
```

Comment:
This can be false if both the "immix_non_moving" and the "sticky_immix_non_moving_nursery" are enabled

Reply:
We should only check against `immix_non_moving`, which disables all kinds of moving. If we don't have `immix_non_moving`, and we have `sticky_immix_non_moving_nursery`, we may still do defrag.